doc/howto.md

```diff
@@ -149,3 +149,84 @@ _ = pd.DataFrame({"x": range(5)}).to_sql("numbers", engine)
 %%sql
 SELECT * FROM numbers
 ```
+
+## Switch connections
+
+```{versionadded} 0.5.2
+`-A/--alias`
+```
+
+```{code-cell} ipython3
+from sqlalchemy import create_engine
+import pandas as pd
+
+engine_one = create_engine("sqlite:///one.db")
```

Comment:
Do we have to create an engine like that with named sessions?

Reply:
no. this part is just loading sample data